test/cmake/seqan3_require_ccache.cmake

```diff
@@ -5,7 +5,7 @@
 # shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
 # -----------------------------------------------------------------------------------------------------
 
-cmake_minimum_required (VERSION 3.10)
+cmake_minimum_required (VERSION 3.15)
```

Comment:
why 15 here?

Reply:
The CMAKE_CXX_COMPILER_LAUNCHER is 3.15, I removed the fallback for pre 3.15